bbj-vscode/src/language/bbj.langium

```diff
@@ -195,6 +196,9 @@ ElseStatement:
 IfEndStatement:
     ('ENDIF' | 'FI') {infer IfEndStatement};
 
+ReturnStatement:
+    ENDLINE_RETURN | 'RETURN' returnValue=Expression?;
```

Comment:
Suggestion: In case the normal `RETURN` keyword is used, `returnValue` cannot be optional. If it isn't there, we always get an `ENDLINE_RETURN` token instead.

Reply:
Ups, thanks.